spring-data-mongodb/src/test/java/org/springframework/data/mongodb/repository/SimpleReactiveMongoRepositoryTests.java

```diff
@@ -441,6 +442,14 @@ public void findOneShouldEmitIncorrectResultSizeDataAccessExceptionWhenMoreThanO
 		StepVerifier.create(repository.findOne(example)).expectError(IncorrectResultSizeDataAccessException.class);
 	}
 
+	@Test // DATAMONGO-1907
+	public void existsByExampleShouldReturnNonExistingWithoutThrowException() {
+
+		Example<ReactivePerson> example = Example.of(new ReactivePerson("foo", "bar", -1));
+
+		StepVerifier.create(repository.findOne(example)).expectNextCount(0).verifyComplete();
```

Comment:
`.expectNextCount(0)` can be omitted and simplified to `StepVerifier.create(repository.findOne(example)).verifyComplete();`